base/tools/src/bulkissuance/CMakeLists.txt

```diff
@@ -33,5 +33,5 @@ install(
     FILES
         bulkissuance.data
     DESTINATION
-        ${SHARE_INSTALL_PREFIX}/pki/native-tools/
+        ${SHARE_INSTALL_PREFIX}/pki/tools/
```

Comment:
I don't find this in my `/usr/share/pki/tools`. This is because, the parent `CMakeLists.txt` does not add this directory: https://github.com/dogtagpki/pki/blob/07036b2b2a87d31c49dc435bcaed2bc211d6a775/base/tools/src/CMakeLists.txt

````
$ ls -la /usr/share/pki/tools/
total 12
drwxr-xr-x.  2 root root 4096 Sep 30 14:10 .
drwxr-xr-x. 23 root root 4096 Sep 30 14:11 ..
-rw-r--r--.  1 root root 2347 Sep 30 13:37 setpin.conf
````

Reply:
Yeah, looks like we're not building/using it anymore. PKI 10.5 doesn't have it either:
https://github.com/dogtagpki/pki/blob/master/base/native-tools/src/CMakeLists.txt
We probably can remove the source in a separate PR.